task9/index.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import {config} from "./config/config.js";
+import meteorRouter from './delivery/meteorController.js'
+import errorHandler from './exception/errorHandler.js';
+
+const app = express();
+
+app.use(meteorRouter, errorHandler)
```

Comment:
this is not a mistake but better to split routers and error handlers

```js
app.use(meteorRouter)
app.use(errorHandler)
```

Reply:
fixed